Entities/MovableObject.h

```diff
@@ -2006,6 +2028,10 @@ enum MOType
 	unsigned int m_LastCollisionSimFrameNumber;
 	// If true, the object will receive OnPieMenu event whenever someone activated a pie menu
 	bool m_ProvidesPieMenuContext;
+    // Update every n frames property
+    unsigned int m_UpdatePeriod;
```

Comment:
This is also using the old comment style. The new one, is different to support autogenerated docs. I'd also change the wording, so it's clearer, and the variable name needs changing as discussed in the issue comments, so it's more obvious and accurate (don't forget to change the getter and setter accordingly):
```
int m_SimUpdatesBetweenScriptedUpdate; //!<  The number of simulation updates that should elapse between each scripted update for this MovableObject.
int m_SimUpdatesSinceLastScriptedUpdate; //!< The number of simulation updates that have elapsed since this object's Lua was last updated.
```

Also, no need to make it unsigned really. Obviously it'll always be positive, but that difference doesn't matter much here and it keeps things simpler.